Resources/Prototypes/_CM14/Catalog/Cargo/cargo_clothing.yml

```
@@ -0,0 +1,39 @@
+- type: cargoProduct
+  id: CMCargoClothingLargePouches
+  icon:
+    sprite: _CM14/Objects/Clothing/Pouches/large_drop.rsi
+    state: icon
+  product: CMCrateClothingLargePouches
+  cost: 2000
+  category: Clothing
+  group: market
+
+- type: cargoProduct
+  id: CMCargoClothingMedicalPouches
+  icon: 
+    sprite: _CM14/Objects/Clothing/Pouches/firstaid.rsi
+    state: icon
+  product: CMCrateClothingMedicalPouches
+  cost: 2000
+  category: Clothing
+  group: market
+
+- type: cargoProduct
+  id: CMCargoClothingWebbing
+  icon:
+    sprite: _CM14/Objects/Clothing/webbing.rsi
+    state: webbing
+  product: CMCrateClothingWebbing
+  cost: 2000
+  category: Clothing
+  group: market
+
+- type: cargoProduct
+  id: CMCargoClothingOfficerOutfit
+  icon:
+    sprite: _CM14/Objects/Clothing/Uniforms/Jumpsuit/ro.rsi
```

Wrong rsi location is causing the check to fail
```suggestion
    sprite: _CM14/Objects/Clothing/Uniforms/ro.rsi
```